cmd/juju/model/destroy.go

```diff
@@ -359,6 +360,30 @@ func newTimedModelStatus(ctx *cmd.Context, api DestroyModelAPI, tag names.ModelT
 			}
 			return nil
 		}
+		isError := func(s string) bool {
+			return corestatus.Error.Matches(corestatus.Status(s))
+		}
+		for _, s := range status {
+			for _, v := range s.Machines {
+				if isError(v.Status) {
+					ctx.Infof("unable to destroy machine: %q, \ngot error: %q\ntry `juju status` for more details.", v.Id, v.Message)
+					return nil
```

Comment:
We don't want to return nil in case a script is checking the error code.
Return ErrSilent